example/package.json

```diff
@@ -4,9 +4,9 @@
   "version": "0.0.0",
   "scripts": {
     "build": "NODE_ENV=production kaliber-build",
-    "watch": "CONFIG_ENV=dev kaliber-watch",
+    "watch": "CONFIG_ENV=dev NODE_OPTIONS='--enable-source-maps' kaliber-watch",
     "serve": "kaliber-serve",
-    "serve.dev": "PORT=8000 CONFIG_ENV=dev kaliber-serve",
+    "serve.dev": "PORT=8000 CONFIG_ENV=dev NODE_OPTIONS='--enable-source-maps' kaliber-serve",
```

Comment:
Concreet: geen source maps op prd, right?

Reply:
In de basis wel, althans, niet in de log files op de server. We kunnen dat, wanneer nodig aanzetten. Ik wil wel kijken of we via de API de source maps naar Rollbar kunnen sturen.

Ohw en voor client side code heb je nog steeds gewoon source maps in productie, deze optie is alleen voor stack traces die door node worden gegenereerd.

Reply:
@larsvankleef Ligt eraan wat je bedoelt met 'goed te laten werken'. Hij doet het prima zonder die `NODE_OPTIONS`, alleen worden de foutmeldingen die door node ge-console-logged worden anders niet door de source maps heen gehaald.

Bij `kaliber-watch` zou je ze eigenlijk in dat bestand bovenin willen zetten (https://github.com/kaliberjs/build/blob/master/library/bin/watch.js#L1), maar dat kan niet.
Voor `kaliber-serve` heb je te maken met het feit dat je het op `dev` wel aan wilt hebben, maar op `prd` niet.